atomicapp/nulecule/container.py

```diff
@@ -85,7 +85,7 @@ def pull(self, image, update=False):
 
         if self.dryrun:
             logger.info("DRY-RUN: %s", pull_cmd)
-        elif subprocess.check_output(pull_cmd) != 0:
+        elif subprocess.check_call(pull_cmd) != 0:
```

Comment:
I think neither the older or new version of this line are going to work. Both ``check_call`` and ``check_output`` raises ``CalledProcessError`` exception when a non-zero status is returned. So, I think we should handle the ``CalledProcessError`` exception and raise ``DockerException`` rather than inside the ``elif`` block.